### DISCUSSION TITLE
🌟 Proud to be at Sabaragamuwa University of Sri Lanka! 🌟

### DISCUSSION THREAD

**Discussion by Kishoby** [community/community#127442] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

I’m studying Computing and Information Systems at this fantastic university. Our beautiful campus in Belihuloya offers a serene environment, perfect for both learning and personal growth. The expert faculty here are leaders in their fields, dedicated to providing an enriching educational experience.

The vibrant student community and diverse clubs make university life exciting and engaging. Our program equips us with cutting-edge tech skills, preparing us for future challenges in the tech industry. 🚀

Join me on this incredible journey! #SabaragamuwaUniversity #CIS #StudentLife #SriLanka

https://www.sab.ac.lk/ 
![pic_1](https://github.com/community/community/assets/168715191/5bc33cf5-a496-41c2-86d3-a1036d728236)

**Comment by ebndev** [comment]
Hi @Kishoby, thanks for being a part of the GitHub Community!

Although we're happy for your accomplishments, the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. This is considered "off-topic" for the Community and as a result, I will be locking this discussion. Please feel encouraged to post again when you have questions in the forum!

Best of luck!
